### ISSUE TITLE
Some resource files in lector/resources/raw/ are installed twice

### ISSUE THREAD

**Issue by guoyunhe** [BasioMeusPuga/Lector#63]
For example, lector.desktop and lector.png has been installed in `/usr/share` but they are also installed in `/usr/lib/`. (tested with 0.3.1)

**Comment by BasioMeusPuga** [comment]
Yeah. This is probably the result of data_files in setup.py. I don't really see this as a major issue though. Is it causing packaging issues?

**Comment by guoyunhe** [comment]
here will be some warning when building RPM package. But it can be solved by replacing with symlink.

For users, only disadvantage is the waste of disk space.

Maybe you can move these files which are installed in /use/share to a different source folder. Then they will not be installed in /use/lib

**Comment by terrycloth** [comment]
I just noticed this issue too. `Lector.png` gets installed to the hicolors icons directory, `lector.desktop` is installed to the desktop applications directory, and after pull request #120, the `.metainfo.xml` file will be installed in `/usr/share/metainfo/` (or similar). So I don't think there's any reason _keep_ them in `./lector/resources/raw/`, at least.

It's a minor problem, though I might have initially assumed it was an easy fix. Is there something about `setup.py` that would make this inconvenient? I haven't done many builds or packages for Python, so I'm too familiar with the gritty details here.

As a packager, having unneeded duplicate files bugs me... but I suppose I probably wouldn't push for it either, if this were going to be a pain in the ass to deal with.
